libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -516,17 +516,36 @@ public boolean isVcSuccessful(List<VcStatusDto> currentVcStatuses, String criIss
                 .getIsSuccessfulVc();
     }
 
-    private boolean isEvidenceVc(VcStoreItem item) throws CredentialParseException {
-        JsonNode vcEvidenceNode = getVCClaimNode(item.getCredential(), VC_EVIDENCE);
+    private boolean isEvidenceVc(String credential, boolean isNonZero)
```

Comment:
Sorry I didn't make this clear - I think we _should_ check for nonzero in all cases (including the existing usage).

If a VC has zero scores, we shouldn't be using it to generate the user identity!